data_module/data_preprocessor.py

```diff
@@ -0,0 +1,86 @@
+import torch
+from torchvision.transforms import v2
+import gensim
+import nltk
+import re
+import contractions
+
+
+class TextPreprocessing():
```

Comment:
Try restructuring the functions so that they resemble this format
def func(a=a,
                b=b,
                c=c)